### PR TITLE
release notes 21.1.1

### PR DESCRIPTION
**Pull request by moishce**
## Status
In Progress

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@moishce Doc review completed

File: content-repo/extra-docs/releases/21.1.1.md [hunk 1]
```diff
@@ -0,0 +1,1060 @@
...
+### Google Drive Pack v1.0.1
+#### Layouts
+- **layout-quickView-Google_Drive_Activity.json** - 
+Maintenance and stability enhancements.
+- **layout-mobile-Google_Drive_Activity.json** - 
+Maintenance and stability enhancements.
+
+---
+
+### Gurucul Risk Analytics Pack v1.0.1 (Partner Supported)
```
Comment (Itay4) [inline content-repo/extra-docs/releases/21.1.1.md:458]: remove this

File: content-repo/extra-docs/releases/21.1.1.md [hunk 2]
```diff
@@ -0,0 +1,1060 @@
...
+#### Integrations
+##### PhishTank v2
+- The *Additional details* URL in the ***create_verified_markdown*** command output is now clickable.
+- Updated the Docker image to: *demisto/python3:3.9.1.14969*.
+
+---
+
+### Phishing Pack v2.0.3
+#### Incident Types
+#####Phishing
```
Comment (Itay4) [inline content-repo/extra-docs/releases/21.1.1.md:761]: fix markdown formatting here

File: content-repo/extra-docs/releases/21.1.1.md [hunk 3]
```diff
@@ -0,0 +1,1060 @@
...
+
+#### Layouts
+**Darktrace Model Breach Layout** - 
+Maintenance and stability enhancements.
+
+---
+
+### Deprecated Content Pack v1.6.7
+#### Classifiers
+##### Palo Alto Networks Cortex
```
Comment (Itay4) [inline content-repo/extra-docs/releases/21.1.1.md:349]: please remove this

**Review by Itay4 — APPROVED** [review]

**Pull request merged**